tests/sbc_tests.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Affero General Public License v3, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+import pytest
+import torch
+from torch import eye, ones, zeros
+from torch.distributions import MultivariateNormal
+
+from sbi.analysis import check_sbc, run_sbc, get_nltp
+from sbi.inference import SNLE, SNPE_C, simulate_for_sbi
+from sbi.simulators import linear_gaussian
+
+
+@pytest.mark.parametrize(
+    "method, model",
+    [
+        (SNPE_C, "mdn"),
+    ],
+)
+def test_running_sbc(method, model):
```

Comment:
Seeding should be setup:
```suggestion
def test_running_sbc(method, model, set_seed):
```

Reply:
not needed because randomness does not affect the test here.